samples/Add Voice Calling/Web/JavaScript/public/index.html

```diff
@@ -0,0 +1,25 @@
+﻿<!-- © Microsoft Corporation. All rights reserved. -->
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
```

Comment:
we shouldn't be loading assets from 3p domains, 
I assume this is for UI - should we switch to Fluent UI ?